k8s/releases/careers/careers-stage.yaml

```diff
@@ -0,0 +1,46 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: careers
+  namespace: careers-stage
+  labels:
+    app: careers
+  annotations:
+    fluxcd.io/automated: "true"
+    filter.fluxcd.io/chart-image: regex:^(stg-[0-9a-f]{6,10})$
+spec:
+  releaseName: careers
+  timeout: 1200
+  chart:
+    repository: https://mozilla-it.github.io/helm-charts/
+    name: careers
+    version: "0.0.2"
+  values:
+    autoscaling:
+      minReplicas: 2
+      maxReplicas: 3
+    image:
```

Comment:
do feel free to test this, but I don't believe flux picks up new images if you don't explicitly include the docker repository in the helmrelease.

we ran into this with an early migration (pastebin, i think, can dig up PRs & convo on it). even if the default in the helm chart Values is the right repo, it wasn't getting picked up in the flux deploy.